Automattic/jetpack
# VideoPress: dont allow editing video data when doesn't belong to site

This PR disables the title and description fields of the Details panel when the video doesn't belong to the site.
Follow-up PR will add a Notice and probably a modal explaining the limitations when creating a block instance by using a video that doesn't belong to the site.

Fixes #
Part of https://github.com/Automattic/jetpack/issues/30417

## Proposed changes:

* VideoPress: dont allow editing video data when the video doesn't belong to the site

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Create/edit a VideoPress video instance
* Set title and description
* Confirm it works as expected

* Create another VideoPress video block that doesn't belong to the site (https://videopress.com/v/NGYE1Hqy)
* Confirm `title` and `description` fields are disabled.


https://user-images.githubusercontent.com/77539/236021584-6e7dc369-32da-4e6a-9bb3-eae21e82b30e.mov
